### DISCUSSION TITLE
The use of depth_hand_refiner in ADetailer

### DISCUSSION THREAD

**Discussion by uzuuzu20002000** [vladmandic/sdnext#2738] [Q&A]
Thank you for always providing us with a great system.

Please tell me one thing.

Because I want to modify the hand automatically,
I get an error when using depth_hand_refiner in ADetailer.
The environment is as follows

「t2t and backend original」
「ADetailer ControlNet model : control_v11f1p_sd15_depth」
「ControlNet module : depth_hand_refiner」

Is there a mistake?

![adetailer](https://github.com/vladmandic/automatic/assets/130671689/850cc74a-156e-43af-aaf0-f64b739f2be5)

**Comment by Woisek** [comment]
You're not using the depth_hand_refiner module, you use the depth_midas module which is, obviously, not what you want. Just switch to it.

**Reply by uzuuzu20002000** [reply]
Thank you very much. I made a mistake with the photo I pasted.

I used depth_hand_refiner module, but got an error.

Are other people able to use it?

Running script postprocess image:                      
                         extensions/adetailer/scripts/!adetailer.py:            
                         ModuleNotFoundError                                    
╭───────────────────── Traceback (most recent call last) ──────────────────────╮
│ /notebooks/automatic/modules/scripts.py:584 in postprocess_image
